terraform/services/cclf-import/main.tf

```diff
@@ -65,8 +65,34 @@ module "cclf_import_queue" {
 
   name = local.full_name
 
-  function_name = module.cclf_import_function.name
-  sns_topic_arn = data.aws_ssm_parameter.bfd_sns_topic_arn.value
+  function_name    = module.cclf_import_function.name
+  policy_documents = [data.aws_iam_policy_document.sns_send_message.json]
+}
+
+data "aws_iam_policy_document" "sns_send_message" {
+  statement {
+    sid     = "SnsSendMessage"
+    actions = ["sqs:SendMessage"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["sns.amazonaws.com"]
+    }
+
+    resources = [module.cclf_import_queue.arn]
+
+    condition {
+      test     = "ArnEquals"
+      variable = "aws:SourceArn"
+      values   = [data.aws_ssm_parameter.bfd_sns_topic_arn.value]
+    }
+  }
+}
+
+resource "aws_sns_topic_subscription" "this" {
+  endpoint  = module.cclf_import_queue.arn
+  protocol  = "sqs"
+  topic_arn = data.aws_ssm_parameter.bfd_sns_topic_arn.value
 }
```

Comment:
I'm seeing here that these changes will result in destroying and remaking the policy and topic subscription: https://github.com/CMSgov/cdap/actions/runs/19314118921/job/55241472316?pr=314

It could be avoided by adding a `moved` block here but I think it's fine to cycle the resources. Nothing stateful in them and we should be able to create the subscription again as long as the queue ARN hasn't changed.